components/patterns/elements/image-loader.tsx

```diff
@@ -0,0 +1,13 @@
+const ImageLoader = ({ src, width }) => {
+  const imageStyles = [
+    {w: 128, s: 'breakpoint_sm_2x'},
+    {w: 384, s: 'breakpoint_md_2x'},
+    {w: 750, s: 'breakpoint_lg_2x'},
+    {w: 1200, s: 'breakpoint_xl_2x'},
+    {w: 3840, s: 'breakpoint_2xl_2x'},
+  ];
+  const style = imageStyles.find(s => s.w === width)?.s || 'breakpoint_xl_2x';
+  // Switch the image style for the desired width. Add the width parameter just to prevent console logs.
```

Comment:
Eh, what console log?

Reply:
Huh, that is rather presumptive of Next and 3rd party image optimization services.